src/main/java/org/jabref/logic/importer/fileformat/PdfMergeMetadataImporter.java

```diff
@@ -48,7 +48,7 @@ public PdfMergeMetadataImporter(ImportFormatPreferences importFormatPreferences)
         this.importFormatPreferences = importFormatPreferences;
```

Comment:
Shouldn't `List<Importer> metadataImporters` be `List<PdfImporter>`?

Reply:
Hah! Nice find! 👍